submission/submitter/src/test/kotlin/ac/uk/ebi/biostd/submission/model/FilesSourceTest.kt

```diff
@@ -48,7 +48,7 @@ class FilesSourceTest(temporaryFolder: TemporaryFolder) {
 
     @Test
     fun `get user file size`() {
-        assertThat(testInstance.size(TEST_USER_FILE)).isGreaterThan(0)
+        assertThat(testInstance.size(TEST_USER_FILE)).isEqualTo(0)
```

Comment:
I am wondering if there is an easy way to create a file of a specific size to test this properly

Reply:
Nice catch. Done.